specification/privatedns/resource-manager/Microsoft.Network/stable/2018-09-01/privatedns.json

```diff
@@ -1624,6 +1624,7 @@
         "minimumTtl": {
           "type": "integer",
           "format": "int64",
+          "x-ms-client-name": "minimumTTL",
```

Comment:
This looks like a breaking change for customers. From what I'm reading it will change the property name in the SDK? Existing API versions should not have their property names changed in the SDK.

Reply:
Seems like minimumTTL is going on the wire, we are getting s360 error for this.

Reply:
But reading this http://azure.github.io/autorest/extensions/#x-ms-client-name it says the name is for use in client code generation, not the wire.

You are breaking the C# SDK for example if you look at the SoaRecord file: https://github.com/AzureSDKAutomation/azure-sdk-for-net/pull/2821/commits/4b72cebb8b76d5dd98f2a9f3793d54d393704c57

Reply:
Hm. Makes sense. Property names should be changed like I was committed initially. I wonder why it's flagging for breaking change then.